templates/cloud/private-cloud.html

```diff
@@ -0,0 +1,483 @@
+		<p>Those are just examples of questions organisations have to ask themselves at the beginning of their private cloud journey. Fortunately, there are <a href="https://microstack.run" class="p-link--external">tools that enable single-node private cloud installation</a> for testing purposes or, in the case of more complex environments, private cloud providers like Canonical offer <a href="/openstack/consulting">consulting services for private cloud deployment</a>. Those usually include existing workloads assessment, architecture guidance, TCO prediction, cloud deployment and workloads migration.</p>
+		<p><a href="/openstack/consulting#get-in-touch">Get in touch with Canonical for private cloud design and delivery&nbsp;&rsaquo;</a></p>
+	</div>
+</section>
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-8">
+			<h2>All you need for your private cloud</h2>
+		</div>
+		<div class="row u-equal-height">
+			<div class="p-card col-6">
+				<div class="p-heading-icon--muted u-vertically-center">
+					<div class="p-heading-icon__header" style="margin-bottom: 0;">
+						<div class="u-hide--small">
+						{{ image (
+							url="https://assets.ubuntu.com/v1/623e36f9-OpenStack_Logo_Mark+%281%29.svg",
+							alt="OpenStack Logo",
+							width="32",
+							height="28",
+							hi_def=True,
+							attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+							loading="lazy"
+							) | safe
+						}}
+						</div>
+            <h4 class="p-card__title" style="margin-top: .3rem">Charmed OpenStack</h4>
+          </div>
+        </div>
+				<hr/>
+				<div class="p-card__content">
+					<p style="margin-top: .5rem" >Charmed OpenStack is a pure upstream OpenStack distribution engineered for the best price-performance with full enterprise support for your production-grade private cloud.</p>
+					<p><a href="/openstack/features">Learn more about Charmed OpenStack&nbsp;&rsaquo;</a></p>
+				</div>
+			</div>
+			<div class="p-card col-6 ">
+				<div class="p-heading-icon--muted u-vertically-center">
+					<div class="p-heading-icon__header" style="margin-bottom: 0;">
+						<div class="u-hide--small">
+						{{ image (
+							url="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg",
+							alt="Ubuntu Logo",
+							width="28",
+							height="28",
+							hi_def=True,
+							loading="lazy",
+							attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+							) | safe
+						}}
+						</div>
+						<h4 class="p-card__title" style="margin-top: .3rem">Ubuntu Server</h4>
+					</div>
+				</div>
+				<hr/>
+				<div class="p-card__content">
+					<p style="margin-top: .5rem">Ubuntu is the most popular Linux distribution across public clouds, data centres and the edge with a built-in security and compliance for enterprise customers.</p>
+					<p><a href="/server">Learn more about Ubuntu Server</a></p>
+				</div>
+			</div>
+		</div>
+	</div>
+</section>
+{% endblock content %}
```

Comment:
```suggestion
{% endblock content %}

```

Reply:
Would it mean insert a blank line before and after `{% endblock content %}`?

Reply:
Just after! It just complains if there isn't an empty line at the end that's all

Reply:
Oh okay I added one before as well. Will remove it!